Mixins generates wrong font file name when unicode-range subset is number

**Describe the bug**
mixins generates wrong font file name so error thrown while building.
It affects every fonts that `$unicodeMap` is mapped by number, especially CJK fonts (`noto-sans-*`, `noto-serif-*`, ...)

My environment is:
 * vue 2.6.12
 * webpack 4.46.0
 * sass (dart-sass) 1.32.6

**To Reproduce**

build with this source
```scss
@import '~@fontsource/noto-sans-kr/scss/mixins';

@include fontFace($weight: 300);
@include fontFace($weight: 400);
@include fontFace($weight: 500);
```

throws error below
```
Syntax Error: ModuleNotFoundError: Module not found: Error: Can't resolve '@fontsource/noto-sans-kr/files/noto-sans-kr-[0]-300-normal.woff2' in '/workspace/packages/app-user/src'
```

actual file name should be `noto-sans-kr-0-300-normal.woff2`.

**Expected behavior**

change index like `[0]` to just numeric like `0` or `'0'` to work.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -23,6 +23,7 @@ These will always contain changes from Fontsource's end.
 ### Fixes
 
 - Removed all default variable flags from SCSS mixins till further notice to resolve errors relating to importing multiple fonts. [#155](https://github.com/fontsource/fontsource/pull/155)
+- Resolved incorrect unicodeMap numeric subset names that linked to the wrong files. [#167](https://github.com/fontsource/fontsource/pull/167)
 
 ### Documentation
 

--- a/scripts/google/run.js
+++ b/scripts/google/run.js
@@ -66,6 +66,7 @@ if (changed || force === "force") {
   // Make the key value pairs in the required format - subset: (unicodeRangeValues), subset:...
   const unicodeMap = Object.entries(font.unicodeRange)
     .map(subArr => {
+      subArr[0] = subArr[0].replace(/[[\]]/g, "")
       subArr[1] = `(${subArr[1]})`
       return subArr.join(": ")
     })

